maven-model/src/test/java/org/apache/maven/model/v4/Xpp3DomPerfTest.java

```diff
@@ -18,22 +18,6 @@
  */
 package org.apache.maven.model.v4;
 
-/*
```

Comment:
I checked. This is hard violation of Apache policy. From https://www.apache.org/legal/src-headers.html

Do not modify or remove any copyright notices or licenses within third-party works.
Make sure that every third-party work includes its associated license, even if that requires adding a copy of the license from the third-party download site into the distribution.
Do not add the standard Apache License header to the top of third-party source files.

Reply:
I am the author and owner of this file, so I gave it to Codehaus, but now I'm giving it to the ASF.
So the pointer you gave does not apply here.

Reply:
We need a lawyer, or at least official approval. This is complicated and Apache is rigorous about crossing the t's and dotting the I's. IANAL and cannot decide on who owns what or what rules apply where outside the common cases, and this one is getting increasingly wonky.

Reply:
Possibly you can ask at https://issues.apache.org/jira/projects/LEGAL/issues/LEGAL-654?filter=allopenissues

I'm fine with whatever they say is fine.